### PR TITLE
[New Rule] Finder Sync Plugin Enabled

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/734

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/persistence_finder_sync_plugin_pluginkit.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "https://github.com/specterops/presentations/raw/master/Leo Pitt/Hey_Im_Still_in_Here_Modern_macOS_Persistence_SO-CON2020.pdf",
+]
+risk_score = 47
+rule_id = "37f638ea-909d-4f94-9248-edd21e4a9906"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan=5s
```
Comment (brokensound77) [inline rules/macos/persistence_finder_sync_plugin_pluginkit.toml:28]: To help reduce scope

```suggestion
sequence by host.id, user.id with maxspan=5s
```

File: rules/macos/persistence_finder_sync_plugin_pluginkit.toml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ [process where event.type in ("start", "process_started") and process.name == "pluginkit" and process.args == "-a"]
+ [process where event.type in ("start", "process_started") and process.name == "pluginkit" and
+  process.args == "-e" and process.args == "use" and process.args == "-i"]
```
Comment (brokensound77) [inline rules/macos/persistence_finder_sync_plugin_pluginkit.toml:31]: nit:

```suggestion
    process.args == "-e" and process.args == "use" and process.args == "-i"]
```

**Review by threat-punter — APPROVED** [review]
👌

File: rules/macos/persistence_finder_sync_plugin_pluginkit.toml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2020/12/18"
+maturity = "production"
+updated_date = "2020/12/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Finder Sync plugins enables users to extend Finder’s functionality by modifying the user interface. Adversaries may
+abuse this feature by adding a rogue Finder Plugin to repeatedly execute malicious payloads as part of persistence.
```
Comment (bm11100) [inline rules/macos/persistence_finder_sync_plugin_pluginkit.toml:10]:
```suggestion
Finder Sync plugins enable users to extend Finder’s functionality by modifying the user interface. Adversaries may
abuse this feature by adding a rogue Finder Plugin to repeatedly execute malicious payloads for persistence.
```

File: rules/macos/persistence_finder_sync_plugin_pluginkit.toml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+   process.args : "-e" and process.args : "use" and process.args : "-i" and 
+   not process.args in 
+                    ("com.google.GoogleDrive.FinderSyncAPIExtension",
+                     "com.google.drivefs.findersync",
+                     "com.boxcryptor.osx.Rednif",
+                     "com.adobe.accmac.ACCFinderSync", 
+                     "com.microsoft.OneDrive.FinderSync", 
+                     "com.insynchq.Insync.Insync-Finder-Integration", 
+                     "com.box.desktop.findersyncext")
+  ]
```
Comment (bm11100) [inline rules/macos/persistence_finder_sync_plugin_pluginkit.toml:40]:
```suggestion
sequence by host.id, user.id with maxspan = 5s
  [process where event.type in ("start", "process_started") and process.name : "pluginkit" and process.args : "-a"]
  [process where event.type in ("start", "process_started") and process.name : "pluginkit" and
    process.args : "-e" and process.args : "use" and process.args : "-i" and
    not process.args :
    (
      "com.google.GoogleDrive.FinderSyncAPIExtension",
      "com.google.drivefs.findersync",
      "com.boxcryptor.osx.Rednif",
      "com.adobe.accmac.ACCFinderSync",
      "com.microsoft.OneDrive.FinderSync",
      "com.insynchq.Insync.Insync-Finder-Integration",
      "com.box.desktop.findersyncext"
    )
  ]
```

**Review by bm11100 — APPROVED** [review]

**Review by brokensound77 — APPROVED** [review]
LGTM 👍

**Pull request merged**